app/services/clipboard/clipboard.ts

```diff
@@ -1,11 +1,12 @@
-import { mutation, StatefulService } from './stateful-service';
-import { ScenesService } from './scenes';
-import { SourcesService } from './sources';
-import { shortcut } from './shortcuts';
-import { Inject } from '../util/injector';
-import { SourceFiltersService } from './source-filters';
+import { mutation, StatefulService } from 'services/stateful-service';
+import { ScenesService } from 'services/scenes';
+import { SourcesService } from 'services/sources';
+import { shortcut } from 'services/shortcuts';
+import { Inject } from '../../util/injector';
```

Comment:
Webpack will recognize any folder name in the `app` directory, so this can alternatively be written as `util/injector`

Reply:
good to know, thanks!